### ISSUE TITLE
Implement cross entropy and other classic loss functions

### ISSUE THREAD

**Issue by PrplHrt** [PrplHrt/deeplearninglibrary#6]
```
#TODO: Implement cross entropy and other classic loss functions
```

undefined
